opennms-webapp/src/main/java/org/opennms/web/utils/Bootstrap.java

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2023 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2023 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.web.utils;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import org.apache.taglibs.standard.lang.support.ExpressionEvaluatorManager;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.PageContext;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+public class Bootstrap {
+
+    private final PageContext pageContext;
+
+    private final List<String> headTitles = Lists.newArrayList();
+
+    private final List<Breadcrumb> breadcrumbs = Lists.newArrayList();
+
+    private String ngApp;
+    private String scrollSpy;
+
+    private Set<String> flags = Sets.newHashSet();
```

Comment:
Guava has it's uses, but it also creates a lot of dependency headaches. Please don't use it if all you're calling is no argument `newBlahBlah()` things.

It's [basically](https://github.com/google/guava/blob/v20.0/guava/src/com/google/common/collect/Lists.java#L86) the [same](https://github.com/google/guava/blob/v20.0/guava/src/com/google/common/collect/Sets.java#L163) amount of typing. :)